### PR TITLE
add get members route

### PR DESCRIPTION
**Pull request by oloveluck**
route for getting all members. Also fixes a problem with the get applicants route that wasn't working properly... oops

### CODE REVIEW

File: api/src/main/java/com/codeforcommunity/dto/team/UsersResponse.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+public class UsersResponse {
+  private Map<Integer, TeamRole> users;
+
+  public UsersResponse(Map<Integer, TeamRole> users) {
+    this.users = users;
+  }
+
+  private UsersResponse() {
+  }
+
```
Comment (florisdobber) [inline api/src/main/java/com/codeforcommunity/dto/team/UsersResponse.java:16]: Getters and setters for users?

File: service/src/main/java/com/codeforcommunity/processor/TeamsProcessorImpl.java [hunk 2]
```diff
@@ -52,6 +53,19 @@ private void checkGoalExists(int goalId) {
     }
   }
 
+  private UsersResponse getUsers(int teamId, TeamRole role) {
```
Comment (florisdobber) [inline service/src/main/java/com/codeforcommunity/processor/TeamsProcessorImpl.java:56]: Nice abstraction!

File: service/src/main/java/com/codeforcommunity/processor/TeamsProcessorImpl.java [hunk 3]
```diff
@@ -346,6 +350,11 @@ public void leaveTeam(JWTData userData, int teamId) {
     }
   }
 
+  @Override
+  public UsersResponse getMembers(int teamId) {
```
Comment (florisdobber) [inline service/src/main/java/com/codeforcommunity/processor/TeamsProcessorImpl.java:354]: Here you also want to return people with a leader role

**Review by florisdobber — COMMENTED** [review]
2 quick fixes, rest looks good!

**Review by connernilsen — APPROVED** [review]
Nice job! I think it'll be ready once you address Floris' comment

**Review by florisdobber — APPROVED** [review]

**Pull request merged**